torch/_tensor_docs.py

```diff
@@ -5433,15 +5441,34 @@ def callable(a, b) -> number
     RuntimeError: Tensor size(-2) 3 needs to be divisible by blocksize[0] 2
     >>> x.to_sparse(layout=torch.sparse_csr, blocksize=(3, 1))
     RuntimeError: to_sparse for Strided to SparseCsr conversion does not use specified blocksize
+
+    >>> x = torch.tensor([[[1], [0]], [[0], [0]], [[2], [3]]])
+    >>> x.to_sparse(layout=torch.sparse_csr, dense_dim=1)
+    tensor(crow_indices=tensor([0, 1, 1, 3]),
+           col_indices=tensor([0, 0, 1]),
+           values=tensor([[1],
+                          [2],
+                          [3]]), size=(3, 2, 1), nnz=3, layout=torch.sparse_csr)
+
 """,
 )
 
 add_docstr_all(
     "to_sparse_csr",
     r"""
-to_sparse_csr() -> Tensor
+to_sparse_csr(dense_dim=None) -> Tensor
+
+Convert a tensor to compressed row storage format (CSR).  Except for
+strided tensors, only works with 2D tensors.  If the :attr:`self` is
+strided, then the number of dense dimensions could be specified, and a
+hybrid CSR tensor will be created.
```

Comment:
It would be worth mentioning what happens with the rest of `self.dim() - 2 - dense_dim` dimensions. I guess these will be batch dimensions..